### PR TITLE
[Asset] Fixed blurry rasterized SVGs

### LINKED ISSUE #10086: [Bug]: SVG image blurred
### Expected behavior

SVG image thumbs are sharp

### Actual behavior

SVG image thumbnails are all blurred.

### Steps to reproduce

Go to pimcore Demo:
https://demo.pimcore.com/admin/login/deeplink?document_151_page

 

Go to the snippet: 
https://demo.pimcore.com/admin/login/deeplink?document_166_snippet
and change the image (need to override) to this svg: https://demo.pimcore.com/admin/login/deeplink?asset_368_image

### PR DESCRIPTION
**Pull request by brusch**
fixes #10086

### CODE REVIEW

**Comment by brusch** [comment]
@iibs can you please check if this PR fixes your issue? Thanks!

File: lib/Image/Adapter/Imagick.php [hunk 1]
```diff
@@ -48,11 +48,17 @@ class Imagick extends Adapter
...
+    private ?array $initalOptions = null;
+
     /**
      * {@inheritdoc}
      */
     public function load($imagePath, $options = [])
     {
+        if(null === $this->initalOptions) {
+            $this->initalOptions = $options;
+        }
```
Comment (jdreesen) [inline lib/Image/Adapter/Imagick.php]:
```suggestion
        $this->initalOptions ??= $options;
```

**Comment by ramundomario** [comment]
@brusch base branch seems wrong since it's added to milestone 10.2.3

**Comment by brusch** [comment]
@ramundomario good catch - thanks!

**Comment by brusch** [comment]
@iibs any feedback? Thanks!

**Pull request merged**